microsetta_admin/templates/email_stats_pulldown.html

```diff
@@ -13,7 +14,11 @@
 
     $(document).ready(function() {
         {% if resource is not none %}
-            $('#search_results').DataTable();
+            $('#search_results').DataTable({
+                dom: 'Bfrtip',
```

Comment:
Can you comment what Bfrtip does?

Reply:
honestly have zero idea... its a code structure for initialization. Detail is here (https://datatables.net/reference/option/dom), i'll add a comment pointing there